modules/openapi-generator/src/main/resources/python-blueplanet/app/{{packageName}}/models/model.mustache

```diff
@@ -52,15 +52,17 @@ class {{classname}}(Model):
 {{/vars}}
 
     @classmethod
-    def from_dict(cls, dikt){{^supportPython2}} -> '{{classname}}'{{/supportPython2}}:
+    def from_dict(cls, dikt, attr_map=True){{^supportPython2}} -> '{{classname}}'{{/supportPython2}}:
```

Comment:
Btw to make it fully backwards compatible shouldn't we use False as default value?

Reply:
Hmm it looks like.from_dict is used in controllers when ingesting json and using it to create a model instance. So this was originally a bug that json payloads were not using the attr map variable names.
Good catch, you are right that this will be a breaking change if we default it to false.

How about adding a generator specific parameter `fromDictUsesAttrMap` and using the value of that parameter in the mustache template? You can default that parameter to false and making it a parameter allows people to generate code with it set to true also.

Reply:
You mean `breaking change if we default it to true` not `false`?
If yes, this looks like a good idea for me. I know nothing about generator specific parameter, can you guide where to look?

Reply:
So you would need to add a clIOption named something like `fromDictUsesAttrMap`.
Here is [a sample PR](https://github.com/OpenAPITools/openapi-generator/commit/3959530465c228668d9a76ad0f8fa8c23358dd6c) that does similar work to add useNose as a CLI Option. CLI Options are described as additional properties that can be passedin when running generators and are documented [here](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/python-blueplanet.md).
Make sure to add fromDictUsesAttrMap as a class variable to the generator class and then I think that you can use it in templates.